.changeset/gold-countries-kneel.md

```diff
@@ -0,0 +1,5 @@
+---
+'@primer/components': major
+---
+
+CircleBadge no longer accepts styled-system props. Please use the `sx` prop to extend Primer component styling instead. See also https://primer.style/react/system-props.
```

Comment:
Should we link to the `sx` prop doc instead?

Reply:
That's reasonable. Maybe I'll include the system props link also if folks are curious about the migration?